pom.xml

```diff
@@ -22,15 +22,15 @@
   <groupId>io.cdap.cdap</groupId>
   <artifactId>cdap-event-writer-extensions</artifactId>
   <name>CDAP Event Writer Extensions</name>
-  <version>1.0.2</version>
+  <version>1.0.3-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <modules>
     <module>cdap-event-writer-ext-gcp-pubsub</module>
   </modules>
 
   <properties>
-    <cdap.version>6.7.2</cdap.version>
+    <cdap.version>6.7.3-SNAPSHOT</cdap.version>
```

Comment:
We generally don't update the CDAP dependencies unless needed.

Reply:
Thanks, removed